host_core/lib/host_core.ex

```diff
@@ -196,8 +196,7 @@ defmodule HostCore do
 
             %{
               config
-              | cluster_seed: config.cluster_seed,
```

Comment:
Does this mean we won't be able to refer to a host_config file to use generated cluster seeds for subsequent hosts?

Reply:
no, this was a bug that basically used to say `config.cluster_seed: config.cluster_seed` which is dum. Now it's using the right value :)